From 187a6da61fae6ed6cd4d8cecf848bcfde0cae383 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 3 Dec 2018 10:09:00 +0800
Subject: [PATCH] arm64: dts: rockchip: fix wrong lcd_en settings on
 rk1808-evb.

GPIO0_C4 is used for lcd_en,not for backlight_en.
So fix it.

Change-Id: Id38ea83d65c91e2d7772d79caa6bad84a5d4398c
Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts | 1 +
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi    | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
index 1f48a7818e8d..572221f11c3c 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
@@ -35,6 +35,7 @@
 		compatible = "sitronix,st7703", "simple-panel-dsi";
 		reg = <0>;
 		backlight = <&backlight>;
+		enable-gpios = <&gpio0 RK_PC4 GPIO_ACTIVE_HIGH>;
 		power-supply = <&vcc5v0_sys>;
 		prepare-delay-ms = <2>;
 		reset-delay-ms = <1>;
diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 3f9bc6bc88fa..9f78c17c57a4 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -70,7 +70,6 @@
 			240 241 242 243 244 245 246 247
 			248 249 250 251 252 253 254 255>;
 		default-brightness-level = <200>;
-		enable-gpios = <&gpio0 RK_PC4 GPIO_ACTIVE_HIGH>;
 	};
 
 	display_subsystem: display-subsystem {
-- 
2.35.3

